doc/classes/HSlider.xml

```diff
@@ -10,6 +10,9 @@
 	<tutorials>
 	</tutorials>
 	<theme_items>
+		<theme_item name="grabber_offset" data_type="constant" type="int" default="0">
+			Vertical grabber offset.
```

Comment:
This sounds like "offset of vertical grabber" while it should be more like "vertical offset of the grabber". Needs rewording for better clarity, same below.